src/utils/http.js

```diff
@@ -65,12 +65,16 @@ export function http(method, url, data, headers = {}) {
     };
 
     if (data) {
-        data = Object.assign({}, data);
-
-        if (method === 'GET') {
-            url = stringifyGETParams(url, data);
-        } else if (method === 'POST' || method === 'PUT') {
-            fetchOptions.body = JSON.stringify(data);
+        if (data instanceof FormData) {
+            fetchOptions.body = data;
+            delete fetchOptions.headers['Content-Type'];
```

Comment:
This is basically to let `fetch` create the header, it adds the form boundary in it. I'll add a comment